astro.config.mjs

```diff
@@ -15,7 +15,7 @@ export default defineConfig({
   site: "https://ddev.com",
   vite: {
     server: {
-      allowedHosts: true, // leave this unchanged for DDEV!
+      allowedHosts: ["." + process.env.DDEV_TLD], // leave this unchanged for DDEV!
```

Comment:
`allowedHosts: true` doesn't work anymore in vite v6:

- https://github.com/vitejs/vite/issues/19242

I saw this when I opened `ddev launch :4321`:

```
Blocked request. This host ("ddev.com.ddev.site") is not allowed.
To allow this host, add "ddev.com.ddev.site" to `server.allowedHosts` in vite.config.js.
```

So I updated it to accept `DDEV_TLD` subdomains.

This is needed for `@astrojs/react` v3 (vite v5) to `@astrojs/react` v4 (vite v6) update.